components/BackButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Head from 'next/head';
+
+const BackButton = () => {
+console.log('BackButton being rendered')
+  return (
+    <div>
```

Comment:
This outer div only complicates the DOM tree since it does nothing but contain another div.